pyrovision/inference/__init__.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) Pyronear contributors.
+# This file is dual licensed under the terms of the CeCILL-2.1 and AGPLv3 licenses.
+# See the LICENSE file in the root of this repository for complete details.
+
+from .pyronear_predictor import PyronearPredictor
```

Comment:
I think we should only be adding a model in `pyrovision.models`
Just like in torchvision (https://github.com/pytorch/vision/blob/master/torchvision/models/resnet.py) we can add the checkpoint URL in a dict at the top of the file, and write a function to instantiate the model (with pretrained as argument)

Reply:
Ok I get your point, I will make a rexnet module